src/NServiceBus.Persistence.AzureStorage/Timeout/AzureStorageTimeoutPersistence.cs

```diff
@@ -9,6 +9,9 @@ namespace NServiceBus
     using Persistence.AzureStorage.Config;
 
     /// <summary></summary>
+    [ObsoleteEx(Message = "Azure transports support timeouts natively and do not require timeout persistence.",
```

Comment:
@distantcam does this obsoletion message look good to you (1 of 2)?

Reply:
Wouldn't it be better to mention the specific transport instead of generic 'azure transports'. Maybe one day we will have a new one that doesn't? And then these statements need to be replaced, causing an uninteded release.

Reply:
No new transport will require timeouts persistence and we're heading towards removing this kind of persistence for delayed delivery (which is timeouts). If Timeouts are to be implemented using persistence, it will unlikely to be the same implementation and would need to be distinguished from the old timeouts.

Old versions of the ASB legacy transport and older versions of ASQ are the azure transports that required this persistence. Are you comfortable to just mention ASQ?

Reply:
yeah just mention ASQ

Reply:
Resolving, messages updated.